### DISCUSSION TITLE
Rearrange mods in mod select

### DISCUSSION THREAD

**Discussion by cluccles** [ppy/osu#24901] [Ideas]
Perhaps (in say difficulty increase) rearrange the mods to be from more commonly used to less commonly used so that you don't have to scan halfway down the mod select for hidden or nightcore, so say at the top would be your basic DT,HD,HR,NC etc and your far less used mods would be to the bottom.

Just a little suggestion :D

**Comment by Rekunan** [comment]
I think this could be solved by having presets of the mods you use the most often.
